backend/routes.js

```diff
@@ -258,7 +267,37 @@ export default class Servers {
                         status = 201;
                     }
                     saveSetting('USERS', existingUsers);
-                    res.json(status, {});
+
+                    if (contains(username, getSetting('ALLOWED_USERS'))) {
+                        res.setCookie('plotly-auth-token', access_token, {'path': '/'});
+
+                        const db_connector_access_token = generateAndSaveAccessToken();
+                        res.setCookie('db-connector-auth-token',
+                                      db_connector_access_token, {
+                                          'maxAge': getAccessTokenExpiry(),
```

Comment:
should this just be `db_connector_acccess_token`?

Reply:
Sorry, dont get it.. you mean the cookie-name should be `db-connector-access-token` instead of `db-connector-auth-token` ..?

Reply:
I actually don't know what I was referring to.